assets/base.css

```diff
@@ -1142,7 +1142,7 @@ summary::-webkit-details-marker {
   }
 
   .grid--peek.slider .grid__item:last-of-type {
-    margin-right: 1.5rem;
+    margin-right: calc(-1 * var(--grid-mobile-horizontal-spacing));
```

Comment:
I haven't investigated yet to see which line is causing this issue, but the thumbnail slider now looks broken. I wonder if it's because we changed from a fixed margin to a variable, while the thumbnails are not using this variable. Will share an update if I find it more details.

Videos below. As you can see, the scrolling is really off when it reaches the last slide.

**Before:**

https://user-images.githubusercontent.com/37168033/169359718-b42b6415-536d-44a9-a97d-f74b19216bd9.mp4

**After**

https://user-images.githubusercontent.com/37168033/169359728-49659935-b691-4381-9e2d-4e46e2ee2aba.mp4

Reply:
Got this fixed. It was because I was applying the fix to it when it's actually not needed in that context. So I'm overwriting it now in `section-main-product.css` with: 
```css
.slider--mobile.thumbnail-list:after {
  content: none;
}
```